extension/aodn-listeners/src/main/java/au/org/emii/listeners/history/AodnAttachmentAddedListener.java

```diff
@@ -0,0 +1,28 @@
+package au.org.emii.listeners.history;
+
+import org.fao.geonet.domain.StatusValue;
+import org.fao.geonet.events.history.AttachmentAddedEvent;
+import org.springframework.context.ApplicationListener;
+import org.springframework.stereotype.Component;
+
+@Component
+public class AodnAttachmentAddedListener extends AodnMetadataEventListener implements ApplicationListener<AttachmentAddedEvent> {
+
+    private String changeMessage = "";
+    private String eventType = StatusValue.Events.ATTACHMENTADDED;
+
+    @Override
+    public String getChangeMessage() {
```

Comment:
getChangeMessage and getEventType don't seem to be required by the interface and aren't used anywhere.   Do we need them?

Reply:
Not used but would be useful if we want to log any messages or event type.

Reply:
Removed unused methods.